README.md

```diff
@@ -1582,6 +1583,36 @@ In your other local project directory:
 yarn link @okta/okta-signin-widget
 ```
 
+### Utilizing Pseudo-loc
+
+> :warning: This tool requires access to Okta's internal registry via the VPN.
+
+A pseudo-localized language is a test language created to identify issues with the internationalization process. Generated from `login.properties` English resources, the pseudo-loc properties file can be used to test UI's for English leaks and CSS layout issues caused due to localization.
+
+To generate pseudo-loc, run the following command:
+
+```sh
+# Navigate into the pseudo-loc package
+[okta-siginin-widget]$ cd packages/@okta/pseudo-loc/
+
+# Install all required dependencies and generate login_ok_PL.propertiies
+# NOTE: This requires VPN access
+[pseudo-loc]$ yarn install
+[pseudo-loc]$ yarn pseudo-loc
+```
+
+Whenever a build, start, or test command is executed, pseudo-loc will run to ensure all current `login.properties` resources are mapped to this test locale.
```

Comment:
Can we be a bit explicit here and call out for Okta developers who are on VPN running start will autogenerated the login_ok_PL.propertiies with any new keys they have added in login.properties.

Reply:
Good idea. I'll be more verbose here.